test/testdata/rewriter/minitest_tables.rb

```diff
@@ -86,13 +86,48 @@ def self.it(name, &blk); end
   end
 
 
-  test_each [1, 2, 3] do |k, v| # error: Wrong number of parameters for `test_each` block
-    it "does not handle more than one argument" do
+  test_each [1, 2, 3] do |k, v|
```

Comment:
Can you add (or are there already) tests for when the first argument to `test_each` isn't a list syntactically?

```ruby
test_each(value) do |x, y|
end
```

Reply:
Are you looking for the combination of a value argument and block with multiple arguments? If not, there is this case:
https://github.com/sorbet/sorbet/blob/077ec005/test/testdata/rewriter/minitest_tables.rb#L51-L57

Reply:
> Are you looking for the combination of a value argument and block with multiple arguments?

yep, specifically multiple block arguments too

Reply:
I added multiple arg/destructuring variants of the existing tests in b54e51fd532a2b1eebb8a06984239da3d115d1d4

Reply:
awesome, thanks!